add support for multiple playlists

sdetweil opened the pull request:
give this a look

all playlist results returned as one big list
the video: number will be wrong now cause it was by url

cowboysdude commented:
trying to resolve this:

 results=[],
    playlist_index=0,
    playlist_loading=false,
    startTimer: function(routine, timeout){
setTimer(()=> {
            routine;
}, timeout);
    }
    },

Giving me an error on the 1st line...

On Thu, Dec 27, 2018 at 5:43 PM sam detweiler <notifications@github.com>
wrote:

> give this a look
>
> all playlist results returned as one big list
> the video: number will be wrong now cause it was by url
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/cowboysdude/Tube/pull/3
> Commit Summary
>
>    - add support for multiple playlists
>
> File Changes
>
>    - *M* node_helper.js
>    <https://github.com/cowboysdude/Tube/pull/3/files#diff-0> (63)
>
> Patch Links:
>
>    - https://github.com/cowboysdude/Tube/pull/3.patch
>    - https://github.com/cowboysdude/Tube/pull/3.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cowboysdude/Tube/pull/3>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKgOEMEnpjueHedG_3TlXotiixByFZa5ks5u9U0hgaJpZM4ZjQiA>
> .
>

sdetweil commented:
sorry, this.results=[];

i did not try to run this, as I have no playlists

cowboysdude commented:
For some reason the node_helper is a bit messed up.   Not sure what
happened in the pull request but it is requesting urls all over the
place... HAAHA

On Thu, Dec 27, 2018 at 6:23 PM sam detweiler <notifications@github.com>
wrote:

> its the = sign, should be colon
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/cowboysdude/Tube/pull/3#issuecomment-450251627>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AKgOEATSuhBrmx8kF0WIRTA8IRZwQXXmks5u9VZigaJpZM4ZjQiA>
> .
>

sdetweil commented:
i added support for a list of playlists, just like your code said...  loads
one, waits a second, if done, loads another... in the playlist from config

On Thu, Dec 27, 2018 at 5:45 PM John Wade <notifications@github.com> wrote:

> For some reason the node_helper is a bit messed up. Not sure what
> happened in the pull request but it is requesting urls all over the
> place... HAAHA
>
> On Thu, Dec 27, 2018 at 6:23 PM sam detweiler <notifications@github.com>
> wrote:
>
> > its the = sign, should be colon
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/cowboysdude/Tube/pull/3#issuecomment-450251627>, or
> mute
> > the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AKgOEATSuhBrmx8kF0WIRTA8IRZwQXXmks5u9VZigaJpZM4ZjQiA
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cowboysdude/Tube/pull/3#issuecomment-450255095>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABbcHsgLUP4YyrB9vk37zhZu_U8_F_qxks5u9VuwgaJpZM4ZjQiA>
> .
>


-- 
Sam

sdetweil commented:
updated code via pr

On Thu, Dec 27, 2018 at 5:47 PM <sdetweil@gmail.com> wrote:

> i added support for a list of playlists, just like your code said...
> loads one, waits a second, if done, loads another... in the playlist from
> config
>
> On Thu, Dec 27, 2018 at 5:45 PM John Wade <notifications@github.com>
> wrote:
>
>> For some reason the node_helper is a bit messed up. Not sure what
>> happened in the pull request but it is requesting urls all over the
>> place... HAAHA
>>
>> On Thu, Dec 27, 2018 at 6:23 PM sam detweiler <notifications@github.com>
>> wrote:
>>
>> > its the = sign, should be colon
>> >
>> > —
>> > You are receiving this because you modified the open/close state.
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/cowboysdude/Tube/pull/3#issuecomment-450251627>,
>> or mute
>> > the thread
>> > <
>> https://github.com/notifications/unsubscribe-auth/AKgOEATSuhBrmx8kF0WIRTA8IRZwQXXmks5u9VZigaJpZM4ZjQiA
>> >
>> > .
>> >
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/cowboysdude/Tube/pull/3#issuecomment-450255095>, or mute
>> the thread
>> <https://github.com/notifications/unsubscribe-auth/ABbcHsgLUP4YyrB9vk37zhZu_U8_F_qxks5u9VuwgaJpZM4ZjQiA>
>> .
>>
>
>
> --
> Sam
>


-- 
Sam
